FordLabs/PeopleMover
# Add matomo events

## Issue
Resolves #525 
## What was done
- [x] Add more events to matomo

- [x] Category: Space Name Action: Add Role
- [ ] Category: Space Name Action: Assign Role - what would this be
- [x] Category: Space Name Action: Edit Role
- [x] Category: Space Name Action: Delete Role - gives id instead of name
- [x] Category: Space Name Action: Edit Space Name
- [x] Category: Space Name Action: Download Report
- [x] Category: Space Name Action: Filter
- [x] Category: Space Name Action: Sort
- [x] Category: Space Name Action: Revert
- [x] Category: Space Name Action: Archive Product
- [x] Category: Space Name Action: Edit Person
- [x] Category: Space Name Action: Add Location Tag
- [x] Category: Space Name Action: Edit Location Tag
- [x] Category: Space Name Action: Delete Location Tag
- [x] Category: Space Name Action: Add Product Tag
- [x] Category: Space Name Action: Edit Product Tag
- [x] Category: Space Name Action: Delete Product Tag

## How to test
1. Run through the actions outlined in the story
2. Ensure all these events are showing up in matomo
